src/CommandingOfficers/BrennerWolves/BrennerDoR.java

```diff
@@ -0,0 +1,76 @@
+package CommandingOfficers.BrennerWolves;
+
+import CommandingOfficers.Commander;
+import CommandingOfficers.CommanderInfo;
+import CommandingOfficers.DeployableCommander;
+import CommandingOfficers.RuinedCommander;
+import Engine.GameScenario;
+import Terrain.MapMaster;
+import UI.UIUtils;
+import Units.*;
+
+public class BrennerDoR extends RuinedCommander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Brenner", UIUtils.BW);
+      infoPages.add(new InfoPage(
+          "Captain of the 12th Battalion. He has an unshakable faith in humanity’s goodness, and will aid any in need.\n"));
```

Comment:
Just armies in need? I assume you just stole this text from somewhere, but it seems amusingly specific, especially since most of the other armies are actively fighting against him.

Reply:
I copied it from the cart. I guess it makes more sense in the context of the campaign there, lol